src/Xenial.Tasty/Commands/TastyCommand.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+
+using Xenial.Delicious.Execution;
+
+namespace Xenial.Delicious.Commands
+{
+    public class TastyCommand
```

Comment:
should be internal for now, cause I don't want to expose those as API surface yet